public/js/main.js

```diff
@@ -9,5 +9,10 @@ $(document).ready(function(){
         return false;
       }
     });
+
+    $('#preview-image').change( function(event) {
+        var imgpath = URL.createObjectURL(event.target.files[0]);
```

Comment:
please check if not choose image : 
**step 1** : you click and choose a image.
**step 2** : you click and not choose image. please set image default again.